core/cas-server-core/src/test/java/org/apereo/cas/config/CasAuthenticationDelegationTestConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package org.apereo.cas.config;
+
+import org.pac4j.core.context.session.SessionStore;
+import org.pac4j.jee.context.session.JEESessionStore;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * This is {@link CasAuthenticationDelegationTestConfiguration}.
+ *
+ * @author Jerome LELEU
+ * @since 7.0.0
+ */
+@TestConfiguration(value = "CasAuthenticationDelegationTestConfiguration", proxyBeanMethods = false)
+public class CasAuthenticationDelegationTestConfiguration {
```

Comment:
You'll need to move this class to the module that actually handles delegation, not here. This is "core".

Reply:
Copy that